package.json

```diff
@@ -23,9 +23,19 @@
     "typescript-sanitizer"
   ],
   "dependencies": {
+    "es6-shim": "^0.35.3",
```

Comment:
as I told last time its user's choice to use es6-shim or target:es6. Shims never should be forced and user should decide if he will use it or not based on his environment. Thats why it should be removed from dependencies.